src/layers/recurrent.jl

```diff
@@ -46,17 +46,28 @@ struct RNNCell{bias, A, B, W, S} <: AbstractExplicitLayer
     init_state::S
 end
 
-function RNNCell((in_dims, out_dims)::Pair{<:Int, <:Int}, activation=tanh; bias::Bool=true,
-                 init_bias=zeros32, init_weight=glorot_uniform, init_state=ones32)
-    return RNNCell{bias, typeof(activation), typeof(init_bias), typeof(init_weight),
+function RNNCell((in_dims, out_dims)::Pair{<:Int, <:Int}, activation=tanh;
+                 use_bias::Bool=true, bias::Union{Missing, Bool}=missing, init_bias=zeros32,
+                 init_weight=glorot_uniform, init_state=ones32)
+    if !ismissing(bias)
```

Comment:
Can you add a msg like https://lux.csail.mit.edu/dev/devdocs/style_guide/#deprecation so that we don't miss out things while deleting things?

Reply:
Add this comment `# Deprecated Functionality (Remove in v0.5)`

Reply:
The constructor does already warn about the deprecated kwarg similar to Dense. The testset contains a nested 'Deprecations' testset. I've now added a message about the new `use_bias` kwarg to the changelog.

Do you want me to move the `Base.depwarn` from src/layers/recurrent.jl to src/deprecated.jl?

Reply:
No everything else is fine. Just add the comment above the `if !dismissing(bias)` like.

Reply:
done